docs/tutorials/findoneandupdate.md

```diff
@@ -51,17 +51,17 @@ Using the `upsert` option, you can use `findOneAndUpdate()` as a find-and-[upser
 [require:Tutorial.*findOneAndUpdate.*upsert]
 ```
 
-<h2 id="raw-result">The `rawResult` Option</h2>
+<h2 id="includeresultmetadata">The `includeResultMetadata` Option<h2 id="rawresult"></h2></h2>
```

Comment:
```suggestion
<h2 id="includeresultmetadata">The <code>includeResultMetadata</code> Option<h2 id="rawresult"></h2></h2>
```

is the double h2 intended for backward compatibility?

Reply:
Yep the double h2 is so `#rawresult` still works